core/model/src/main/java/org/activityinfo/model/type/period/PeriodValue.java

```diff
@@ -0,0 +1,182 @@
+package org.activityinfo.model.type.period;
+/*
+ * #%L
+ * ActivityInfo Server
+ * %%
+ * Copyright (C) 2009 - 2013 UNICEF
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the 
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public 
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+
+import org.activityinfo.model.resource.IsRecord;
+import org.activityinfo.model.resource.Record;
+import org.activityinfo.model.type.FieldTypeClass;
+import org.activityinfo.model.type.FieldValue;
+
+import java.io.Serializable;
+
+/**
+ * @author yuriyz on 01/27/2015.
+ */
+public class PeriodValue implements FieldValue, IsRecord, Serializable {
+
+    private int year;
+    private int month;
+    private int week;
+    private int day;
+    private int hour;
+    private int minute;
+    private int second;
+
+    public PeriodValue() {
+    }
+
+    public PeriodValue(int year, int month, int week, int day, int hour, int minute, int second) {
```

Comment:
@akbertram in this code I've sticked to `java.util.Date` because there is nice `com.google.gwt.user.datepicker.client.CalendarUtil` which helps to manipulate the dates. Do you think we should use `LocalDate` instead? Then we should *invent* own CalendarUtil.